src/vmspawn/vmspawn-util.c

```diff
@@ -0,0 +1,341 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "architecture.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "json.h"
+#include "log.h"
+#include "macro-fundamental.h"
+#include "memory-util.h"
+#include "path-lookup.h"
+#include "path-util.h"
+#include "sort-util.h"
+#include "string-util-fundamental.h"
+#include "string-util.h"
+#include "recurse-dir.h"
+#include "strv.h"
+#include "vmspawn-util.h"
+#include <stdio.h>
+
+OvmfConfig* ovmf_config_new(void) {
+        return new0(OvmfConfig, 1);
```

Comment:
This is trivial enough to spell out instead of creating such wrapper.

Reply:
Yeah thats fair I will replace with new0 in the required places.

Reply:
Fixed in f8c2832218.